testutils/store_gossiper/store_gossiper.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Matt Tracy (matt@cockroachlabs.com)
+// 		   Bram Gruneir (bram+code@cockroachlabs.com)
+
+package storeGossiper
```

Comment:
Convention seems to be just mashing words together, all lowercase with no delimiters. Mixed-case definitely looks odd; I can't think of anywhere else that I've seen a mixed-case package name and on first read I thought `storeGossiper` was a global variable. Personally I like underscores for this, but golint complains. I also prefer directory names and package names to match, so I would rename the directory (and file) from `store_gossiper` to whatever we settle on.

`gossiputil.StoreGossiper`?